templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -220,6 +220,24 @@
             },
             "Retry": ${retry_statement},
             "ResultPath": null,
+            "Next": "Insert into post-ingest table"
+          },
+          "Insert into post-ingest table": {
```

Comment:
Maybe "Insert asset into post-ingest table"?

Reply:
I'm going to leave it as is because that might sound like we're inserting the whole asset. You can always inspect the step definition if you want to see what it's inserting.